features/admin.actions.v1/components/pre-issue-access-token-action-config-form.tsx

```diff
@@ -281,113 +283,117 @@ const PreIssueAccessTokenActionConfigForm: FunctionComponent<PreIssueAccessToken
     };
 
     return (
-        <RulesProvider
-            conditionExpressionsMetaData={ RuleExpressionsMetaData }
-            initialData={ actionData?.rule }
-        >
-            <FinalForm
-                onSubmit={ (values: ActionConfigFormPropertyInterface, form: any) => {
-                    handleSubmit(values, form.getState().dirtyFields); }
-                }
-                validate={ validateForm }
-                initialValues={ initialValues }
-                render={ ({ handleSubmit, form }: FormRenderProps) => (
-                    <form onSubmit={ handleSubmit }>
-                        <EmphasizedSegment
-                            className="form-wrapper"
-                            padded={ "very" }
-                            data-componentid={ `${ _componentId }-section` }
-                        >
-                            <div className="form-container with-max-width">
-                                { renderFormFields() }
-                                { !isLoading && (
-                                    <Button
-                                        size="medium"
-                                        variant="contained"
-                                        onClick={ handleSubmit }
-                                        className={ "button-container" }
-                                        data-componentid={ `${ _componentId }-primary-button` }
-                                        loading={ isSubmitting }
-                                        disabled={ getFieldDisabledStatus() }
-                                    >
-                                        {
-                                            isCreateFormState
-                                                ? t("actions:buttons.create")
-                                                : t("actions:buttons.update")
+        <>
+            { !isActionLoading && actionData && (
```

Comment:
Shall we include error as well in this condition?

Reply:
Updated.